apps/src/blockly/themes/cdoDark.js

```diff
@@ -9,9 +9,9 @@ export default GoogleBlockly.Theme.defineTheme(Themes.DARK, {
   base: DarkTheme,
   blockStyles: cdoBlockStyles,
   componentStyles: {
-    workspaceBackgroundColour: color.neutral_dark90,
-    blackBackground: color.neutral_dark80,
-    flyoutBackgroundColour: color.neutral_dark70,
+    workspaceBackgroundColour: color.music_black,
```

Comment:
Nit: if this color is used across all labs, maybe rename this color to be more general?

Reply:
Good point.  I learned this morning that there will probably be a new round of colour names, so I think it might be worth updating this as soon as they are decided upon.  @levadadenys might have more information too.

Reply:
I guess `dark_black` should be the best name for this color (#121212), cause that's how it's called in figma/design system token for now.
In case we'll need it in the future: `dark` - stands for dark theme, `black` - stands for black color.
Let me know if you want to see it in Figma and I'll send you the link. (Not sure if it's ok to share figma links in GitHub pr)

Reply:
Updated.  Thanks!

Reply:
I find it interesting that `black` is darker than `dark_black` 😁
Thanks for the update!